src/js/internal/util/inspect.js

```diff
@@ -453,11 +436,16 @@ try {
 }
 
 const extractedSplitNewLines = value => {
-  if (typeof value === "string") {
-    return extractedSplitNewLinesFastPathStringsOnly(value) || extractedSplitNewLinesSlow(value);
+  const lines = StringPrototypeSplit(value, "\n");
```

Comment:
`pop()` is unnecessary and will potentially cause the array to resize. Reading the `lines.length` property is only needed once. Checking if the line is the last line on each iteration is unnecessary. 

```suggestion
  const lines = StringPrototypeSplit(value, "\n");
  let i = 0;
  for (let j = 0, length = lines.length; j < length; j++) {
      const line = lines[j];
     if (line !== "")
          lines[i++] = line + "\n";
  }
  lines.length = i;
  return lines;
```